stdlib/string.ncl

```diff
@@ -202,9 +202,9 @@
       "%
     = fun s => %str_chars% s,
 
-    code | CharLiteral -> Num
+    codepoint | Character -> Num
     | doc m%%"
-      Results in the ascii code of the given character.
+      Results in the unicode codepoint of the given character if it fits into a single codepoint.
```

Comment:
I'm not entirely sure of the correctness of this function and `from_codepoint`. But I do think that this is the interface we should be aiming for and if it doesn't work quite right at the moment that should be a bug in the evaluator.